.flake8

```diff
@@ -2,4 +2,11 @@
 import-order-style = google
 application-import-names = e2c, tests
 per-file-ignores =
-    tests/*: D104
+    # D104 Missing docstring in public package
+    tests/__init__.py: D104,
+    tests/e2c/__init__.py: D104,
+ignore =
+    # E203 whitespace before ':'
+    E203,
```

Comment:
You should not disable global E203. Instead, use `# noqa: E203` for specific lines.

Reply:
Okay. [E203](https://www.flake8rules.com/rules/E203.html) is import indeed. But it conflicts with `black` when writing dictionaries. Is there any better solution?

Reply:
It only conflicts in this case and flake8 is wrong here. It's useful in other cases.

Reply:
Got it. Will apply ignorance to those correct lines.